src/ERC721Drop.sol

```diff
@@ -463,10 +463,25 @@ contract ERC721Drop is
         onlyPublicSaleActive
         returns (uint256)
     {
-        return _handlePurchase(quantity, comment);
+        return _handlePurchase(msg.sender, quantity, comment);
     }
 
-    function _handlePurchase(uint256 quantity, string memory comment) internal returns (uint256) {
+    /// @notice Purchase a quantity of tokens to a specified recipient, with a comment
+    /// @param recipient recipient of the tokens
+    /// @param quantity quantity to purchase
+    /// @param comment comment to include in the IERC721Drop.Sale event
```

Comment:
should we leave a comment to leave this blank if you do not wish to leave a comment?